source/include/platforms/machine_amiga_mos.hpp

```diff
@@ -24,14 +24,14 @@ namespace ExVM {
 
     public:
       void      set()     {
-        ::ReadEClock(&mark);
+        ReadEClock(&mark);
```

Comment:
I was a little surprised by this. It suggests that ReadEClock may not be an actual function but some sort of macro construct?